src/utils/message_builder.rs

```diff
@@ -1,6 +1,8 @@
 use std::default::Default;
 use std::fmt::{self, Write};
 use ::model::{ChannelId, Emoji, Mentionable, RoleId, UserId};
+use std::ops::Add;
+use std::convert::{From, Into};
```

Comment:
These are a part of the [stdlib prelude](https://doc.rust-lang.org/std/prelude/), explicitly bringing them into scope is redundant.